_includes/working-with-tables.md

```diff
@@ -0,0 +1,208 @@
+<a name="quick-help"/>
+
+## Quick Help
+
+1. In the _Preparation_ menu on the left sidebar, click on **Libraries**.
+1. At the top left of the _Libraries_ table, click **Receive**.
+1. In the dialog box, enter:{% if include.detailed == true %}
+    * Aliquot Class: **gDNA (aliquot)**{% endif %}
+    * Quantity: **5**
+1. Click **Receive**.
+
+The Quick Help section at the top of the page contains instructions for how to interact with multiple table cells at once. 
+
+1. Click the **Quick Help** header at the top right of the page, under the save button. 
+    * To close the Quick Help section, click again on the **Quick Help** header. 
+
+## Bulk Data Entry
+
+<a name="copy-paste"/>
+
+### Copy & Paste
+If you have a spreadsheet that already has the sample information and you need to enter this into MISO, copy-pasting is often the fastest way to do this. 
+
+1. Open the _Quick Help_. Note which key combination your current computer uses to paste. 
+1. Copy the items from the table below (do not copy the "Sample Type" cell).
+1. Click in the first row of the _Sample Type_ column, and use the key combination to paste.
+
+| Sample Type |
+|-------------|
+|GENOMIC|
+|GENOMIC|
+|GENOMIC|
+|GENOMIC|
+|GENOMIC|
+
+### Fill many
+Entering the same data in all rows can be done by using the "fill from once cell" feature. Enter the data in one cell, then you can fill in multiple cells by:
+  * double-clicking the square in the bottom right corner of that cell to fill in all cells below.
+  * clicking the square in the bottom right corner of that cell, then dragging the mouse up or down the selected number of rows to fill those cells with the initial value.
+
+<a name="fill-down-drag"/>
+
+#### Drag (fills up or down)
```

Comment:
* The h4 style stands out a lot more than the h3 style... Probably better to use only 3 levels of headers.
* The exercises are numbered in the other tutorials, so probably should be in this one as well